src/Footer.tsx

```diff
@@ -148,6 +172,52 @@ export const Footer = memo(
                 style={style}
                 {...rest}
             >
+                {linkList !== undefined && (
+                    <div className={fr.cx("fr-footer__top")}>
+                        <div className={fr.cx("fr-container")}>
+                            <div
+                                className={fr.cx(
+                                    "fr-grid-row",
+                                    // "fr-grid-row--start", // why is this class used in dsfr doc?
```

Comment:
Doc isn't up to date. We know what class are and aren't in the CSS because we have type safety, they don't.